CHANGELOG.md

```diff
@@ -1,5 +1,41 @@
 # Delta Chat Android Changelog
 
+## v1.0.0
+2019-12-17
+
+Finally, after months of coding and fixing bugs, here it is: Delta Chat 1.0 :)
+An overview over the changes since v0.510:
+
+* a new core; for better stability, speed and future maintainability,
+  the core is written completely in the Rust programming language now
+* for end-to-end-encryption, rPGP is used now;
+  the rPGP library got a first independent security review mid 2019
+* improved behavior of sending and receiving messages in flaky networks
```

Comment:
i suggest to put this part further down -- not sure how much we really are better -- i think the move to async has, for now, rather lead to some regressions.

Reply:
moved it down.